data/signersDefault.json

```diff
@@ -325,6 +325,18 @@
 			"sdid": "buyon.com",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL"
+		}, {
+			"domain": "cacf-festgeld.de",
+			"addr": "*",
+			"sdid": "cacf-festgeld.de",
+			"ruletype": "NEUTRAL",
+			"priority": "DEFAULT_RULE_NEUTRAL"
+		}, {
+			"domain": "cacf-festgeld.de",
+			"addr": "*@service.cacf-festgeld.de",
+			"sdid": "service.cacf-festgeld.de",
+			"ruletype": "ALL",
+			"priority": "DEFAULT_RULE_ALL"
```

Comment:
I know it may be a little confusing only looking at the names of the priorities.

But the first one for `*` should have `DEFAULT_RULE_ALL` (2100), and the second one for `*@service.cacf-festgeld.de` should be `DEFAULT_RULE_ALL_2` (2110).

Currently with the first one being more general and with a higher priority of `DEFAULT_RULE_NEUTRAL` (2200), it would always be selected other the second one.

See also https://github.com/lieser/dkim_verifier/wiki/Sign-rules#priority and https://github.com/lieser/dkim_verifier/blob/bcd50af59c8b56025c27b1666b001ee19937d23c/modules/dkim/signRules.mjs.js#L58-L67